### PR TITLE
punctuation, some superscript numerals & others

### PR DESCRIPTION
**Pull request by wollmers**

### CODE REVIEW

File: gt/eval/ONB_ibn_19110701_012/ONB_ibn_19110701_012.tif_tl_2.gt.txt [hunk 1]
```diff
@@ -1 +1 @@
-Innsbrucker Nachrichten
\ No newline at end of file
+Jnnsbrucker Nachrichten
```
Comment (stweil) [inline gt/eval/ONB_ibn_19110701_012/ONB_ibn_19110701_012.tif_tl_2.gt.txt]: As "I" and "J" look identical in Fraktur, "Innsbrucker" might be the better transcription. Ideally the neural network could learn when to recognize "J" or "I".

**Review by stweil — CHANGES_REQUESTED** [review]

File: gt/eval/ONB_ibn_19110701_012/ONB_ibn_19110701_012.tif_tl_2.gt.txt (hunk 1 above)
Comment (wollmers) [reply gt/eval/ONB_ibn_19110701_012/ONB_ibn_19110701_012.tif_tl_2.gt.txt]: Changed J -> I in Fraktur now.

File: gt/train/ONB_aze_18950706_8/ONB_aze_18950706_8.jpg_line_1546496895959_11.gt.txt [hunk 2]
```diff
@@ -1 +1 @@
-im Jn- und Auslanda
\ No newline at end of file
+im Jn- und Auslande
```
Comment (stweil) [inline gt/train/ONB_aze_18950706_8/ONB_aze_18950706_8.jpg_line_1546496895959_11.gt.txt]:
```suggestion
im In- und Auslande
```

Comment (wollmers) [reply gt/train/ONB_aze_18950706_8/ONB_aze_18950706_8.jpg_line_1546496895959_11.gt.txt]: Changed them and other typesetting errors,

**Comment by stweil** [comment]
Great, thank you.

**Pull request merged**